Makefile

```diff
@@ -57,23 +57,32 @@ else
 	@$(REBAR) eunit
 endif
 
+.dialyzer_plt:
+	dialyzer --build_plt -r deps \
+		--apps erts kernel stdlib inets crypto public_key ssl xmerl \
+		--fullpath \
+		--output_plt .dialyzer_plt
+
 check:
 ifeq ($(REBAR_VSN),2)
 	@$(REBAR) compile
-	dialyzer --verbose --no_check_plt --no_native --fullpath \
+	$(MAKE) .dialyzer_plt
+	dialyzer --no_check_plt --fullpath \
 		$(CHECK_FILES) \
-		-Wunmatched_returns \
```

Comment:
These two changes (replicated further below) were performed so that rebar 2 -based analysis is done with the same configuration as rebar3 -based analysis, thus yielding the same results.